### PR TITLE
change to handle no error code

### PR DESCRIPTION
**Pull request by sameerbag**
### Problem Description
No error code was not handle

### Solution Description
Added an error code using which datadog can represent that error code in the graph.

### CODE REVIEW

File: collectors/okta/okta_collector.js [hunk 1]
```diff
@@ -83,6 +84,10 @@ class OktaCollector extends PawsCollector {
         });
     }
 
+    _isNoErrorCode(error) {
+        return error.errorCode ? error.errorCode : "ale00011";
```
Comment (rakhimundhada15) [inline collectors/okta/okta_collector.js:88]: Could you please check do we get errorCode field in error object or it is error.status as mentioned  at 
 Throttling https://github.com/alertlogic/paws-collector/pull/234/files#diff-7857e3061ff43ce02f1e5856dda622d45520533da0adbdc3509b3d98d4b74031R92

**Comment by rakhimundhada15** [comment]
Also add the Unit test case of this.

File: collectors/okta/okta_collector.js (hunk 1 above)
Comment (sameerbag) [reply collectors/okta/okta_collector.js]: Added status code instead of errorcode

Comment (sameerbag) [reply collectors/okta/okta_collector.js]: Added test case for the same

**Review by kkuzmin — CHANGES_REQUESTED** [review]
Please bump package version

**Comment by sameerbag** [comment]
@kkuzmin : Done, bumped package.json version

**Review by kkuzmin — APPROVED** [review]

**Pull request merged**